TeamSnapSDK/SDK/DataTypes/Messages/TSDKMessagingPermission.h

```diff
@@ -0,0 +1,26 @@
+//
+//  TSDKMessagingPermission.h
+//  TeamSnapSDK
+//
+// Copyright (c) 2015 TeamSnap. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+#import "TSDKCollectionObject.h"
+#import "TSDKObjectsRequest.h"
+
+@interface TSDKMessagingPermission : TSDKCollectionObject
+
+@property (nonatomic, weak) NSString *_Nullable message; //Example:
```

Comment:
IMO, I think the property specifier for nullability reads a bit better. - `(nonatomic, weak, nullable)`

Reply:
I agree with this. I don't think our header generation function takes this into account, however.

Reply:
i'll fix this file, and the header generator.